### DISCUSSION TITLE
Anni precedenti

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#542] [Idee e proposte]
Sarebbe possibile integrare un menu a tendina riportante gli anni scolastici in modo che se una persona è interessata ad esempio a consultare voti,argomenti,note ,circolari,avvisi,assenze,etc dell'anno precendete lo può fare tranquillamente ?

**Comment by giadaschiavone** [comment]
Ciao Luigi, questa cosa si potrebbe fare, ma non credo si farà poiché già al passaggio del nuovo anno, tutti i documenti vengono archiviati e conservati (es. pagelle, circolari, registri etc.). Poi dipende anche dal fatto che al passaggio al nuovo anno se tu hai archiviato o no i documenti e i registri. Spero di esserti stata d'aiuto!

**Comment by trinko** [comment]
Ho già risposto nell'altra [discussione](https://github.com/iisgiua/giuaschool/discussions/543)
